test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,573 @@
+	}
+	if !subnetsExist {
+		newErr.rdsSubnetGroupsError = append(newErr.rdsSubnetGroupsError, "rds subnet group does not contain expected subnets")
+	}
+
+	return newErr
+}
+
+// verify that the subnet groups for elasticache are created
+func verifyCacheSubnetGroups(cacheSvc *elasticache.ElastiCache, name string, subnets []*string) error {
+	newErr := &networkConfigTestError{
+		cacheSubnetGroupsError: []string{},
+	}
+
+	// get elasticache subnet groups by subnet group name
+	describeCacheGroups, err := cacheSvc.DescribeCacheSubnetGroups(&elasticache.DescribeCacheSubnetGroupsInput{
+		CacheSubnetGroupName: aws.String(name),
+	})
+	if err != nil {
+		newErr.cacheSubnetGroupsError = append(newErr.cacheSubnetGroupsError, fmt.Sprintf("error describing elasticache subnet groups: %v", err))
+		return newErr
+	}
+
+	// expect 1 subnet group
+	cacheSubnetGroups := describeCacheGroups.CacheSubnetGroups
+	if len(cacheSubnetGroups) != 1 {
+		newErr.cacheSubnetGroupsError = append(newErr.cacheSubnetGroupsError, fmt.Sprintf("unexpected number of elasticache subnet groups: %d", len(cacheSubnetGroups)))
+	}
+
+	// ensure all subnets exist in subnet group
+	subnetsExist := true
+	for _, subnet := range cacheSubnetGroups[0].Subnets {
+		if !contains(subnets, subnet.SubnetIdentifier) {
+			subnetsExist = false
+			break
+		}
+	}
+	if !subnetsExist {
+		newErr.cacheSubnetGroupsError = append(newErr.cacheSubnetGroupsError, "elasticache subnet group does not contain expected subnets")
+	}
+
+	return newErr
+}
+
+// verify that the peering connection we create has the correct requester info
+func verifyPeeringConnection(session *ec2.EC2, clusterTag, expectedCidr, vpcID string) (*ec2.VpcPeeringConnection, error) {
+	newErr := &networkConfigTestError{
+		peeringConnError: []string{},
+	}
+
+	// filter the peering connections by name and integreatly tag
+	peeringConn, err := session.DescribeVpcPeeringConnections(&ec2.DescribeVpcPeeringConnectionsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(peeringTagName)},
+			},
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.peeringConnError = append(newErr.peeringConnError, fmt.Sprintf("could not describe peering connections: %v", err))
+		return nil, newErr
+	}
+
+	// expect 1 peering connection to be found
+	conns := peeringConn.VpcPeeringConnections
+	if len(conns) != 1 {
+		newErr.peeringConnError = append(newErr.peeringConnError, fmt.Sprintf("unexpected number of vpc peering connections: %d", len(conns)))
+		return nil, newErr
+	}
+
+	// verify that the requester info is correct
+	conn := conns[0]
+	if aws.StringValue(conn.RequesterVpcInfo.CidrBlock) != expectedCidr && aws.StringValue(conn.RequesterVpcInfo.VpcId) != vpcID {
+		newErr.peeringConnError = append(newErr.peeringConnError, fmt.Sprintf("unexpected accepter vpc cidr block: %d", len(conns)))
+	}
+
```

Comment:
Should we add a check to verify the state of the peering connection. We expect `VpcPeeringConnectionStateReasonCodeActive` if it is not active I think we should fail and report back the state of the peering connection. 
Again like above the connection metric should be failing on this, but having the test error would help in debugging the issue IMO